geoportal/Dockerfile

```diff
@@ -36,7 +36,8 @@ COPY geoportailv3_geoportal/static-ngeo /app/geoportailv3_geoportal/static-ngeo
 RUN rm -rf /usr/lib/node_modules/ngeo
 RUN mv /app/geoportailv3_geoportal/static-ngeo/ngeo /usr/lib/node_modules/ngeo
 
-COPY . /app
+RUN mkdir -p /app
+COPY ./package.json /app
 
 # jsapi generation
 ADD ./jsapi /etc/apiv4/
```

Comment:
I think we can get rid of the diff of the `Dockerfile` completely, as the `npm install` in between the two changes only concerns the JS API, if I see correctly. The `package.json` for the app is actually already copied in line 28.

Reply:
the main point of this diff is removing the `COPY . /app` and putting it after the jsapi build
but yes, we can remove 
```
RUN mkdir -p /app
COPY ./package.json /app
```

Reply:
Ok I don't get the advantage of moving `COPY . /app`, neither as the `WORKDIR` changes, but I might be missing something.

Reply:
By moving the cp . as late as possible we avoid all unnecessary docker build steps (npm install for jsapi and build jsapi) if the source code changes
This is just a build optimization which may reduce the image build time if something changes in app

We can leave the Dockerfile as is if that is more comfortable for you.

Reply:
Alright, I'm fine with moving the `COPY . /app`